src/replication.cc

```diff
@@ -25,7 +25,7 @@ FeedSlaveThread::~FeedSlaveThread() {
 Status FeedSlaveThread::Start() {
   try {
     t_ = std::thread([this]() {
-      Util::ThreadSetName("feed-slave-thread");
+      Util::ThreadSetName("feed-slave");
```

Comment:
Let's avoid `slave` from outside observation
```suggestion
      Util::ThreadSetName("feed-replica");
```